src/main/java/components/Event.java

```diff
@@ -0,0 +1,26 @@
+package components;
+
+/**
+ * Class for Event object.
+ */
+public class Event extends Task {
+    protected String from;
+    protected String to;
+
+    /**
+     * Constructor for an Event object.
+     * @param msg Description of the event.
+     * @param from The start date of the event.
+     * @param to The end date of the event.
+     */
+    public Event(String msg, String from, String to) {
```

Comment:
Similar to above, perhaps a more descriptive name could be used, like fromDate and toDate?
Also, I think it would be a good idea to give a complete name "message" instead of "msg" to go with the coding standard.